documentation/troubleshooting/modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -152,3 +152,14 @@ NOT* approve them, therewise you will face even more issues.
 # If you see Subject Name: localhost.localdomain remove the csr
 [kni@provisioner ~]$ oc delete csr <wrong_csr>
 ----
+
+== Deployment failures due to VIP's being used by rogue nodes 
```

Comment:
Incorrect title. I think the issue here is just VRRP conflict

== Deployment failures due to VRRP conflict when rogue nodes are using same cluster name